container-registry/container-analysis/src/main/java/com/example/containeranalysis/CreateOccurrence.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.containeranalysis;
+
+// [START containeranalysis_create_occurrence]
+import com.google.cloud.devtools.containeranalysis.v1beta1.GrafeasV1Beta1Client;
+import com.google.containeranalysis.v1beta1.NoteName;
+import com.google.containeranalysis.v1beta1.ProjectName;
+import io.grafeas.v1beta1.Occurrence;
+import io.grafeas.v1beta1.Resource;
+import io.grafeas.v1beta1.vulnerability.Details;
+import java.io.IOException;
+import java.lang.InterruptedException;
+
+public class CreateOccurrence {
+  // Creates and returns a new Occurrence associated with an existing Note
+  public static Occurrence createOccurrence(String resourceUrl, String noteId, 
+      String occProjectId, String noteProjectId) throws IOException, InterruptedException {
+    // String resourceUrl = "https://gcr.io/project/image@sha256:foo";
+    // String noteId = "my-note";
+    // String occProjectId = "my-project-id";
+    // String noteProjectId = "my-project-id";
+    final NoteName noteName = NoteName.of(noteProjectId, noteId);
```

Comment:
If that's the new sample style fine, but it seems confusing to have both of them.

Reply:
by both of them, do you mean occProjectId and noteProjectId?

Reply:
Passing in the values in the function and the commented lines as well.

Reply:
Yeah it occurred to me that it might be better at this point to have noteName and occProjectName as the parameters to the function. But I think I'd prefer to keep it as is to be consistent with other languages, and avoid a massive refactor. 

I'd be open to removing the commented lines, but I want to make sure I'm following the new sample style as well